src/__tests__/openapiNodegen_exampleFiles.ts

```diff
@@ -0,0 +1,38 @@
+import fs from 'fs-extra';
+import path from 'path';
+import openapiNodegen from '@/generateIt';
+import { clearTestServer, tplUrl } from './helpers';
+
+jest.setTimeout(60 * 1000); // in milliseconds
+const testServerPath = path.join(process.cwd(), 'test_server');
+const swaggerFilePath = path.join(process.cwd(), 'test_swagger.yml');
+
+describe('e2e testing for example files', () => {
+  beforeAll(() => {
+    clearTestServer();
+  });
+  afterAll(() => {
+    clearTestServer();
+  });
+
+  const generateItConfig = {
+    dontRunComparisonTool: false,
+    dontUpdateTplCache: true,
+    updateDependenciesFromTpl: false,
+    mockServer: false,
+    swaggerFilePath,
+    targetDir: testServerPath,
+    template: tplUrl,
+    variables: {
+      name: 'Generate-it Typescript Server'
+    }
+  };
+
+  it('Build, delete the expected example file, regen, the example should stay dead.', async () => {
+    await openapiNodegen(generateItConfig);
+    const exampleFilePath = path.join(testServerPath, 'src/EXAMPLE_app.ts');
```

Comment:
not super important, but might be good to assert that it existed from the gen:
`expect(fs.pathExistsSync(exampleFilePath)).toBe(true);`